docs/comparison.md

```diff
@@ -0,0 +1,439 @@
+    If you are cropping with odd numbers, you will need to change the depth to 16-bit.
+
+
+```py
+## Cropping: Removes letterboxing (black bars) [16-bit required for odd numbers]
+clip1 = core.std.Crop(clip1, left=240, right=240, top=0, bottom=0)
+clip2 = core.std.Crop(clip2, left=0, right=0, top=276, bottom=276)
+clip3 = core.std.Crop(clip3, left=0, right=0, top=21, bottom=21)
+```
+
+!!! note
+    Make sure to check for variable aspect ratios throughout the file and only crop the smallest border.
+
+#### Scaling
+
+Downscales or upscales the video. *This should be used to match sources that have differing resolutions.*
+
+- For upscaling (e.g. 720p -> 1080p), use `EwaLanczos`:
+
+```py
+from vskernels import EwaLanczos
+
+## Upscaling: Increases the resolution of clips to match the highest resolution using EwaLanczos (equivalent scaling to mpv's high-quality profile); recommended
+clip1 = EwaLanczos.scale(clip1, 1920, 1080, sigmoid=True)
+clip2 = EwaLanczos.scale(clip2, 1920, 1080, sigmoid=True)
+clip3 = EwaLanczos.scale(clip3, 3840, 2160, sigmoid=True)
+```
+
+- For downscaling (e.g. 2160p/4K -> 1080p), use `Hermite`:
+
+```py
+from vskernels import Hermite
+
+## Downscaling: Decreases the resolution of clips to match the lowest resolution using Hermite (equivalent scaling to mpv's high-quality profile); not recommended
+clip1 = Hermite.scale(clip1, 1920, 1080, linear=True)
+clip2 = Hermite.scale(clip2, 1920, 1080, linear=True)
+clip3 = Hermite.scale(clip3, 3840, 2160, linear=True)
+```
+
+!!! warning
+    Downscaling is generally not recommended. We suggest upscaling your sources to match the highest resolution unless you have a specific reason (e.g. comparing how a higher resolution file would look on a lower resolution display).
+
+#### Trimming
+
+Removes the first *n* frames from the source. For example, `[24:]` will skip the first 24 frames and start the source at frame 25. *This should be used on sources that are out of sync.*
+
+To get the frame difference, find a unique frame (e.g. scene changes) in the correct and incorrect source. Note the frame numbers each one begin at, then set the difference of the two for the incorrect source.
+
+```py
+## Trimming: Trim frames to match clips (calculate the frame difference and enter the number here)
+clip1 = clip1[0:]
+clip2 = clip2[24:]
+clip3 = clip3[0:]
+```
+
+!!! note
+    For more advanced trimming such as chaining, splicing, and looping, see [Vapoursynth's docs](https://www.vapoursynth.com/doc/pythonreference.html#slicing-and-other-syntactic-sugar).
+
+
+#### Depth
+
+Converts clips to 16-bit depth with 4:4:4 chroma subsampling. *Required for filters such as cropping (with odd numbers) or tonemapping.*
+
+```py
+## Depth: Convert clips to 16-bit 4:4:4 [required for cropping with odd numbers or tonemapping]
+clip1 = core.resize.Bicubic(clip1, format=vs.YUV444P16)
+clip2 = core.resize.Bicubic(clip2, format=vs.YUV444P16)
+clip3 = core.resize.Bicubic(clip3, format=vs.YUV444P16)
```

Comment:
Note that this does not match mpv's behavior (which also uses `ewa_lanczossharp` for chroma on `profile=high-quality`), but then again neither does vspreview's internal chroma scaler when given a yuv420 clip. So I don't know what the best way to handle this is (I guess to fully match mpv, *all* clips should be resized to 444 using ewa_lanczos or vspreview should be configured to do that?) but I wanted to point it out.